source/logger.js

```diff
@@ -0,0 +1,60 @@
+if(Meteor.isServer)
+  winston = Npm.require('winston');
+
+Space.Logger = Space.Object.extend(Space, 'Logger', {
+
+  _logger: null,
+  _state: 'stopped',
+
+  Constructor: function(){
+
+    if(Meteor.isServer) {
+      this._logger = new (winston.Logger)({
+        transports: [
+          new (winston.transports.Console)()
+        ]
+      });
+    }
+    if(Meteor.isClient) {
+      this._logger = console
+    }
+  },
```

Comment:
Now that i see it in code i think we should just add a `Space.base` module that maps these things and let the higher-level modules require it. It seems illogical to circumvent the biggest advantage of Space in its base package :stuck_out_tongue: Then the logger class could just be a simple singleton that is configured like normal.